### PR TITLE
Small bugfixes

### PR DESCRIPTION
**Pull request by BenCat07**

### CODE REVIEW

**Comment by BenCat07** [comment]
debug.engine-pred-others was not meant to be used at all, pp.draw should be used instead

File: src/prediction.cpp [hunk 1]
```diff
@@ -45,69 +43,12 @@ bool PerformProjectilePrediction(CachedEntity *target, int hitbox)
...
-            }
-        }
-}
-
 Vector Predict(Vector &pos, Vector &vel, Vector acceleration, std::optional<float> grounddistance, float &time)
 {
     PROF_SECTION(PredictNew);
     Vector result = pos;
-    // Quadratic + linear function to map position to time (v+a*t)*t
+    result += (acceleration / 2.0f) * pow(time, 2) + vel * time;
```
Comment (TotallyNotElite) [inline src/prediction.cpp:50]: https://youtu.be/-AQfQFcXac8?t=134

File: src/prediction.cpp [hunk 2]
```diff
@@ -118,10 +59,9 @@ Vector PredictStep(Vector pos, Vector &vel, Vector acceleration, std::pair<Vecto
 {
     PROF_SECTION(PredictNew)
     Vector result = pos;
-    // Quadratic + linear function to go one step into the future
 
+    result += (acceleration / 2.0f) * pow(steplength, 2) + vel * steplength;
```
Comment (TotallyNotElite) [inline src/prediction.cpp:63]: https://youtu.be/-AQfQFcXac8?t=134

File: src/prediction.cpp (hunk 1 above)
Comment (BenCat07) [reply src/prediction.cpp:50]: i mean sure i can also not use pow() if you want

**Pull request merged**